### ISSUE TITLE
PeoplePicker: No screen alert about auto-suggestion after the suggestion list is shown up

### ISSUE THREAD

**Issue by MirandaGe** [OfficeDev/office-ui-fabric-react#1013]
### Bug Report

- __Package version(s)__: any
- __Browser and OS versions__: IE 11.0.9600.18538 (haven't tested other version)
- JAWS version: 18.0.2324 and 17.0.2619 (haven't tested other version)

#### Describe the issue:

1. enter some letter like 'a' into input area of PeoplePicker in demo page
2. the auto-suggestion list pop up and the string is resolved as the first people in the list

#### Actual behavior:

There is no screen alert for the suggestion list.

#### Expected behavior:

There should be screen alert for suggestion list.
I tried JAWS on chrome and it works. That's strange.

### If applicable, please provide a codepen repro:

I tried on Fabric React demo page: https://dev.office.com/fabric#/components/peoplepicker
